crates/ilp-cli/src/interpreter.rs

```diff
@@ -12,22 +12,22 @@ use url::Url;
 pub enum Error {
     // Custom errors
     #[error("Usage error")]
-    UsageErr(&'static str),
+    Usage(&'static str),
     #[error("Invalid protocol in URL: {0}")]
-    ProtocolErr(String),
+    Protocol(String),
     // Foreign errors
     #[error("Error sending HTTP request: {0}")]
-    SendErr(#[from] reqwest::Error),
+    Send(#[from] reqwest::Error),
     #[error("Error receving HTTP response from testnet: {0}")]
-    TestnetErr(reqwest::Error),
+    Testnet(reqwest::Error),
     #[error("Error altering URL scheme")]
-    SchemeErr(()), // TODO: should be part of UrlError, see https://github.com/servo/rust-url/issues/299
+    Scheme(()), // TODO: should be part of Urlor, see https://github.com/servo/rust-url/issues/299
```

Comment:
```suggestion
    Scheme(()), // TODO: should be part of UrlError, see https://github.com/servo/rust-url/issues/299
```

Probably accidential? The linked issue is still open, so looking good.